### PR TITLE
change: updating the auto suggest options

### PR DESCRIPTION
**Pull request by Singh-Ish**
### What type of PR is this? (check all applicable)

- [ ] New component
- [ ] New feature
- [ ] Refactor
- [x] Bug fix
- [ ] Optimization / Build Tools
- [ ] Documentation Update

### Is this a breaking change?

If yes, please note which project will be impacted.

- [ ] Yes
- [x] No

### Have you made an entry in the Changelog?

If not, please make an entry or your PR will be declined. Add new entries under the "Unreleased" section of the changelog.

- [x] Yes
- [ ] No

### Have you made changes to dependencies?

If you've made changes to dependencies please note them here, adding if they've been added, removed or updated, including packages version where applicable.

- [ ] Added
- [ ] Updated
- [ ] Removed
- [ ] Combo of any of the above
- [x] No

### QA Instructions, Screenshots, Recordings

Please replace this line with instructions on how to test your changes, and if applicable add a note
about any UI changes or if a test should be done on a specific browser. Add any other detail relevant to this PR.

### UI accessibility concerns?

If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. If there are no accessibility concerns delete this section.

### CODE REVIEW

File: lib/components/Form/AutoSuggest/AutoSuggest.tsx [hunk 1]
```diff
@@ -12,21 +12,25 @@ interface Option {
...
   const [field, meta] = useField(name)
   const { setFieldValue } = useFormikContext<unknown>()
+
   const errorClass = useErrorClass(name)
 
   const [selectedOption, setSelectedOption] = useState<Option | null>(null)
 
-  useEffect(() => {
-    const initialOption = options?.find((option) => option.value === field.value)
+  console.log(field.value)
```
Comment (mcorkum) [inline lib/components/Form/AutoSuggest/AutoSuggest.tsx:27]: Console log

File: lib/components/Form/AutoSuggest/AutoSuggest.tsx [hunk 2]
```diff
@@ -12,21 +12,25 @@ interface Option {
...
 
+  useEffect(() => {
+    const initialOption = options?.find((option) => option.value === meta?.initialValue)
     setSelectedOption(initialOption || null)
-  }, [field.value, options])
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (mcorkum) [inline lib/components/Form/AutoSuggest/AutoSuggest.tsx:32]: Is there a way to do this without eslint disable? Code likes this makes me nervous for reloads.

File: lib/components/Form/AutoSuggest/AutoSuggest.tsx [hunk 3]
```diff
@@ -40,12 +44,14 @@ export const AutoSuggest = ({ ...props }: AutoSuggestProps) => {
...
       options={options}
-      value={selectedOption}
+      value={selectedOption || null}
       onChange={(option: Option | null) => {
         setFieldValue(name, option?.value)
         setSelectedOption(option)
+        if (rest?.onChange) {
+          rest?.onChange(option?.value)
+        }
       }}
```
Comment (mcorkum) [inline lib/components/Form/AutoSuggest/AutoSuggest.tsx:54]: Was onBlur removed on purpose?

File: lib/components/Form/AutoSuggest/AutoSuggest.tsx (hunk 2 above)
Comment (Singh-Ish) [reply lib/components/Form/AutoSuggest/AutoSuggest.tsx:32]: i am fixing this , trying to change the way it is functioning

File: lib/components/Form/AutoSuggest/AutoSuggest.tsx (hunk 1 above)
Comment (Singh-Ish) [reply lib/components/Form/AutoSuggest/AutoSuggest.tsx:27]: removed already , pushing changes

File: lib/components/Form/AutoSuggest/AutoSuggest.tsx (hunk 3 above)
Comment (Singh-Ish) [reply lib/components/Form/AutoSuggest/AutoSuggest.tsx:54]: yeah this was removed on purpose

**Pull request merged**